src/ui/mathquill-editor/MathQuillEditorLoadable.js

```diff
@@ -16,7 +16,7 @@ class MathQuillEditorShimmer extends React.PureComponent {
 }
 
 const MathQuillEditorLoadable = Loadable({
-  loader: () => import( './MathQuillEditor'),
+  loader: () => import(/* webpackChunkName: "czi_prosemirror_mathquill" */ './MathQuillEditor'),
```

Comment:
Could you please use multi-lines as the [official document](https://webpack.js.org/api/module-methods/#magic-comments) recommended?

We might want to use different comments such as

```
import(
  /* webpackChunkName: "czi_prosemirror_mathquill" */
  /* webpackMode: "lazy" */
  './MathQuillEditor'
),
```

Reply:
sounds like a job for prettier 😉